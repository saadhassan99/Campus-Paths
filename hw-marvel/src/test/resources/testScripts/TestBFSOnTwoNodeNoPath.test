# Test BFS on TwoNodeNoPath graph

# build the graph
LoadGraph TwoNodeNoPath TwoNodeNoPath.tsv

# test BFS order

# find the shortest path NodeA to NodeB
FindPath TwoNodeNoPath NodeA NodeB

# find the shortest path NodeB to NodeA
FindPath TwoNodeNoPath NodeB NodeA