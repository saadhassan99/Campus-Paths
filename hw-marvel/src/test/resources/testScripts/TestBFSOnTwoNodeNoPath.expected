# Test BFS on TwoNodeNoPath graph

# build the graph
loaded graph TwoNodeNoPath

# test BFS order

# find the shortest path NodeA to NodeB
path from NodeA to NodeB:
no path found

# find the shortest path NodeB to NodeA
path from NodeB to NodeA:
no path found